packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -44,8 +44,17 @@ const availableViews = [
 	},
 ];
 
-function ViewTypeMenu( { view, onChangeView } ) {
-	const activeView = availableViews.find( ( v ) => view.type === v.id );
+function ViewTypeMenu( { view, onChangeView, supportedViewTypes } ) {
```

Comment:
I only have naming suggestions:

- Using "layout" instead of "view type" everywhere:
  - We expose it as "Layout" in the UI, so it's better to be coherent with that – it also communicates intent better than type, IMO.
  - Some changes would be: `ViewTypeMenu` to `LayoutMenu`, `availableViews` to `availableLayouts`, etc.
  - I understand there's previous code, so unrelated changes could be done separately, not in this PR.
- `supportedViewTypes` prop to `layouts`. Introduced in this PR, so we can do it here already.